src/site/apt/troubleshooting.apt

```diff
@@ -0,0 +1,14 @@
+Troubleshooting
+
+ Standard project-build-plugin errors
+
+*-----*------*------*
+<<Error Message>>		|		<<Cause>> 		| 		<<Possible Fix>>
+*-----*------*------*
+ InvocationTargetException: No class loader avaible for bundle: ch.ivyteam.ivy.dataclasses.build    | Maven uses wrong Java version	|	Change your Java version if you have multiple versions (export JAVA_HOME=</your/Jre/Dir>)  
+*-----*------*------*
+ adittionalVmOptions not getting initialized														| Multiple additionalVmOptions | Make sure you're on the latest version of project-build-plugin if it doesn't help and you have multiple adittionalVmOptions add a whitespace at the end of each argument
```

Comment:
- typo: in title (additional) 
- be specific: it's a known issue of project-build-plugin 7.0.0 and earlier. can be fixed by updating to 7.0.? .

Reply:
As of now (12.09.2019) none of 7.0.X (7.0.4 being newest) versions have fixed the issue with additionalVmArguments